Gemfile.lock

```diff
@@ -172,10 +193,11 @@ DEPENDENCIES
   rake (~> 13.0.1)
   rspec (~> 3.9.0)
   rspec-sidekiq (~> 3.0.3)
+  rubocop-shopify
   shoulda-matchers (~> 4.2.0)
   sidekiq (~> 5.2.3)
   slack-notifier (~> 2.3.2)
   sqlite3 (~> 1.4.2)
 
 BUNDLED WITH
-   2.1.4
+   2.2.15
```

Comment:
Safe?

Reply:
Yeah that's fine, it's just bundler that was updated. I've dockerized the gem for development in #3 
The ruby image comes with bundler `2.2.15`